src/components/Nav.jsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
- [ ] Kindly avoid disabling the linter since it enforces rules for quality code. Instead, you should fix the raised issues.